xml/planning.xml

```diff
@@ -163,4 +163,61 @@
    For registering an already installed system, see <xref linkend="sec.yast-software.register"/>.
   </para>
  </sect1>
+ 
+ <sect1 xml:id="sec.planning.leanos">
+  <title>Changes in installation from &sls; version &product-ga;</title>
+  <para>
+   Starting with &productname; 15, all products are distributed using a single
+   set of installation media.
+  </para>
+  <para>
+   The products available on these unified media are as follows:
```

Comment:
The list of base products is also mentioned in the Installation Quick Start (see https://susedoc.github.io/doc-sle/develop/SLES-installquick/single-html/#sec.sle.installquick.install.lang): "With the Installer you can install all SUSE Linux Enterprise base products: [...]". Please match (for the future: let's move the itemizedlist to an entity and reuse in the Install Quick and Deployment Guide).

Reply:
That's a different list. I don't know which is authoritative, nor the shorthand names, which are useful in AutoYast and other places. Do you know of anywhere or anyone to ask?

Reply:
I would say that the one in the Install Quick is kind of 'authoritative', because it reflects what customers see in the 'Language, Keyboard and Product Selection' screen that appears during installation, see https://susedoc.github.io/doc-sle/develop/SLES-installquick/single-html/images/install_product_sles.png. Therefore I would advise to use it. However, your sentence about the restriction (not all products are supported on all architectures) makes sense. You can also add the shorthand names to the list if you have them (I agree that they are useful for AY, for example).

Reply:
I have checked the list, including the official long and short package names, with Joachim Werner * Jiři Srain. They are now included. The list matches this one, but with more info.